geomfn: simplify InEpsilon checks in tests

otan opened the pull request:
These checks were used repeatedly and copy-pasta'd, why not simplify
them into the one.

Release justification: non-production code change

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/61525)
<!-- Reviewable:end -->

andyyang890 commented on pkg/geo/geomfn/geomfn_test.go:
```diff
@@ -127,3 +129,49 @@ func TestRemoveConsecutivePointsFromGeomT(t *testing.T) {
+}
+
+// geometryInEpsilon  points and ensures that are within the provided epsilon.
+func geometryInEpsilon(t *testing.T, expected, got geo.Geometry, epsilon float64) {
```
nit: maybe rename to `requireGeometryEqualWithinEpsilon`?

andyyang890 commented on pkg/geo/geomfn/geomfn_test.go:
```diff
@@ -127,3 +129,49 @@ func TestRemoveConsecutivePointsFromGeomT(t *testing.T) {
+	geomTInEpsilon(t, expectedT, gotT, epsilon)
+}
+
+func geomTInEpsilon(t *testing.T, expectedT, gotT geom.T, epsilon float64) {
```
nit: maybe rename to `requireGeomTEqualWithinEpsilon`

andyyang890 commented on pkg/geo/geomfn/geomfn_test.go:
```diff
@@ -127,3 +129,49 @@ func TestRemoveConsecutivePointsFromGeomT(t *testing.T) {
 	}
 }
+
+func geometryFromGeomT(t *testing.T, g geom.T) geo.Geometry {
```
nit: maybe rename to `requireGeometryFromGeomT`

andyyang890 commented on pkg/geo/geomfn/geomfn_test.go:
```diff
@@ -127,3 +129,49 @@ func TestRemoveConsecutivePointsFromGeomT(t *testing.T) {
+	return ret
+}
+
+// geometryInEpsilon  points and ensures that are within the provided epsilon.
```
consider rewording?

andyyang890 commented on pkg/geo/geomfn/affine_transforms_test.go:
```diff
@@ -530,15 +523,8 @@ func TestRotateWithPointOrigin(t *testing.T) {
 				require.EqualError(t, err, tt.wantErrStr)
 			} else {
 				require.NoError(t, err)
+				geometryInEpsilon(t, geometryFromGeomT(t,tt.wantGeom), got, 1e-5)
```
should this be in the same scope as the previous code (i.e. outside the else block)?

otan replied:
this one was deliberate -- we should never assert the value if an error occurs (unless it was part of the contract)

otan commented:
bors r=andyyang890

otan commented:
bors r-

otan committed:
geomfn: simplify InEpsilon checks in tests

These checks were used repeatedly and copy-pasta'd, why not simplify
them into the one.

Release justification: non-production code change

Release note: None

otan commented:
bors r=andyyang890

otan commented:
bors r=andyyang890
